api/turing/imagebuilder/imagebuilder.go

```diff
@@ -288,12 +293,23 @@ func (ib *imageBuilder) createKanikoJob(
 		fmt.Sprintf("--context=%s", ib.imageBuildingConfig.KanikoConfig.BuildContextURI),
 		fmt.Sprintf("--build-arg=MODEL_URL=%s", artifactURI),
 		fmt.Sprintf("--build-arg=BASE_IMAGE=%s", baseImage),
+		fmt.Sprintf("--build-arg=MLFLOW_ARTIFACT_STORAGE_TYPE=%s", ib.artifactServiceType),
```

Comment:
can you extract out the kaniko build argument code in separate function?

Reply:
Alright, I just refactored all the credentials-related modifications to kaniko args, volumes, etc. into separate functions (just like the changes I did for Merlin. I didn't create a helper function to add non-credential related build arguments though.